beacon_node/execution_layer/src/engine_api/http.rs

```diff
@@ -926,19 +930,43 @@ impl HttpJsonRpc {
                     .await?;
                 Ok(JsonGetPayloadResponse::V3(response).into())
             }
+            ForkName::Base
+            | ForkName::Altair
+            | ForkName::Bellatrix
+            | ForkName::Capella
+            | ForkName::Electra => Err(Error::UnsupportedForkVariant(format!(
+                "called get_payload_v3 with {}",
+                fork_name
+            ))),
```

Comment:
We usually match all fork names so that we don't miss potential changes to functions on addition of new forks.
For e.g. if ForkName::F* had some changes that affect this function, then we might potentially miss it in a refactor if we do.the if let pattern. 
It won't affect this specific function, but I think its good to be consistent.

Reply:
I think we've kinda shifted policy here towards being less verbose, or at least considering it on a case by case basis. In this case it makes sense that any fork post-electra shouldn't be calling this.

Reply:
we can leave this one since this is in line with all the other similar methods in this file. But we can come back and change them all to reduce boiler plate